core/cli/README.md

```diff
@@ -0,0 +1,101 @@
+# Axon CLI
+
+Axon CLI is a command line tool for Axon framework. It is included in the Axon binary with the command of `cargo build --release`.
+
+## Usage
+
+### Generate Keypair
+
+```bash
+./target/release/axon generate-keypair -n 1
```

Comment:
```suggestion
mkdir -p ./devtools/chain/data
./target/release/axon generate-keypair -n 1 --path ./devtools/chain/data
```

fix `Error writing private key: No such file or directory (os error 2)``

Reply:
The path has the default value `/free-space/` don't need to `mkdir`.

Reply:
> fix `Error writing private key: No such file or directory (os error 2)``

I tried the cmd `./target/release/axon generate-keypair -n 1` in fresh machine, and it threw the above error.

Reply:
Because the folder does not have `free-space` direction.